sdk/types/prices.ts

```diff
@@ -16,6 +16,11 @@ export type Prices<T = Wei> = Record<string, Price<T>>;
 
 export type PricesMap<T = Wei> = Partial<Record<AssetKey, T>>;
 
+export type SynthRate = {
```

Comment:
I was thinking more specifically `SynthPrice` rather than `rate`, as that's the naming convention we shifted to in the sdk, so would be good to maintain consistency there.